index.js

```diff
@@ -40,7 +40,7 @@ module.exports = function markdown(config) {
     }
 
     var md = new Remarkable(options);
-    var ctx = merge({}, options, this.context, context);
+    var ctx = merge({}, options, this, context);
     return md.render(options.fn(ctx));
```

Comment:
thanks! I think this should be `var ctx = merge({}, options, this, this.context, context);`

the `this.context` variable is added by some renderers, like assemble